diff --git a/@bin/format-apply.bash b/@bin/format-apply.bash
index e6bae37..d8ad648 100755
--- a/@bin/format-apply.bash
+++ b/@bin/format-apply.bash
@@ -4,14 +4,8 @@ set -euo pipefail
 
 # detect ROOT_DIR - BEGIN
 SCRIPT_PATH="$([ -L "$0" ] && readlink "$0" || echo "$0")"
-SCRIPT_DIR="$(
-  cd "$(dirname "${SCRIPT_PATH}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${SCRIPT_DIR}/.." || exit 1
-  pwd -P
-)"
+SCRIPT_DIR="$(cd "$(dirname "${SCRIPT_PATH}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${SCRIPT_DIR}/.." || exit 1; pwd -P)"
 # detect ROOT_DIR - END
 
 DEBUG=${DEBUG:-0}
diff --git a/@bin/format-check.bash b/@bin/format-check.bash
index 3aefdf6..071b332 100755
--- a/@bin/format-check.bash
+++ b/@bin/format-check.bash
@@ -4,14 +4,8 @@ set -uo pipefail
 
 # detect ROOT_DIR - BEGIN
 SCRIPT_PATH="$([ -L "$0" ] && readlink "$0" || echo "$0")"
-SCRIPT_DIR="$(
-  cd "$(dirname "${SCRIPT_PATH}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${SCRIPT_DIR}/.." || exit 1
-  pwd -P
-)"
+SCRIPT_DIR="$(cd "$(dirname "${SCRIPT_PATH}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${SCRIPT_DIR}/.." || exit 1; pwd -P)"
 # detect ROOT_DIR - END
 
 DEBUG=${DEBUG:-0}
diff --git a/test/commands/test_command_install_babashka.bats b/test/commands/test_command_install_babashka.bats
index 9890dbe..06b4c27 100755
--- a/test/commands/test_command_install_babashka.bats
+++ b/test/commands/test_command_install_babashka.bats
@@ -8,14 +8,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_clj_kondo.bats b/test/commands/test_command_install_clj_kondo.bats
index f3c7250..5aad6be 100755
--- a/test/commands/test_command_install_clj_kondo.bats
+++ b/test/commands/test_command_install_clj_kondo.bats
@@ -8,14 +8,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_yq.bats b/test/commands/test_command_install_yq.bats
index e18685f..fd9d3b7 100755
--- a/test/commands/test_command_install_yq.bats
+++ b/test/commands/test_command_install_yq.bats
@@ -8,14 +8,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/scripts/test_clone_git_repo.bats b/test/scripts/test_clone_git_repo.bats
index dd885ec..cbfade1 100755
--- a/test/scripts/test_clone_git_repo.bats
+++ b/test/scripts/test_clone_git_repo.bats
@@ -1,14 +1,8 @@
 #!/usr/bin/env bats
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
