diff --git a/test/commands/test_command_install_babashka.bats b/test/commands/test_command_install_babashka.bats
index 9890dbe..06b4c27 100755
--- a/test/commands/test_command_install_babashka.bats
+++ b/test/commands/test_command_install_babashka.bats
@@ -8,14 +8,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_clj_kondo.bats b/test/commands/test_command_install_clj_kondo.bats
index f3c7250..5aad6be 100755
--- a/test/commands/test_command_install_clj_kondo.bats
+++ b/test/commands/test_command_install_clj_kondo.bats
@@ -8,14 +8,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_sosh.bats b/test/commands/test_command_install_sosh.bats
index f9aea4f..49de8da 100755
--- a/test/commands/test_command_install_sosh.bats
+++ b/test/commands/test_command_install_sosh.bats
@@ -7,14 +7,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_yq.bats b/test/commands/test_command_install_yq.bats
index e18685f..fd9d3b7 100755
--- a/test/commands/test_command_install_yq.bats
+++ b/test/commands/test_command_install_yq.bats
@@ -8,14 +8,8 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(
-  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
-  pwd -P
-)"
-ROOT_DIR="$(
-  cd "${TEST_DIR}/../.." || exit 1
-  pwd -P
-)"
+TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
+ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
 # detect ROOT_DIR - end
 
 setup() {
