diff --git a/test/commands/test_command_install_babashka.bats b/test/commands/test_command_install_babashka.bats
index 06b4c27..9890dbe 100755
--- a/test/commands/test_command_install_babashka.bats
+++ b/test/commands/test_command_install_babashka.bats
@@ -8,8 +8,14 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
-ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
+TEST_DIR="$(
+  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
+  pwd -P
+)"
+ROOT_DIR="$(
+  cd "${TEST_DIR}/../.." || exit 1
+  pwd -P
+)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_clj_kondo.bats b/test/commands/test_command_install_clj_kondo.bats
index 5aad6be..f3c7250 100755
--- a/test/commands/test_command_install_clj_kondo.bats
+++ b/test/commands/test_command_install_clj_kondo.bats
@@ -8,8 +8,14 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
-ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
+TEST_DIR="$(
+  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
+  pwd -P
+)"
+ROOT_DIR="$(
+  cd "${TEST_DIR}/../.." || exit 1
+  pwd -P
+)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_sosh.bats b/test/commands/test_command_install_sosh.bats
index 49de8da..f9aea4f 100755
--- a/test/commands/test_command_install_sosh.bats
+++ b/test/commands/test_command_install_sosh.bats
@@ -7,8 +7,14 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
-ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
+TEST_DIR="$(
+  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
+  pwd -P
+)"
+ROOT_DIR="$(
+  cd "${TEST_DIR}/../.." || exit 1
+  pwd -P
+)"
 # detect ROOT_DIR - end
 
 setup() {
diff --git a/test/commands/test_command_install_yq.bats b/test/commands/test_command_install_yq.bats
index fd9d3b7..e18685f 100755
--- a/test/commands/test_command_install_yq.bats
+++ b/test/commands/test_command_install_yq.bats
@@ -8,8 +8,14 @@
 ###
 
 # detect ROOT_DIR - BEGIN
-TEST_DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1; pwd -P)"
-ROOT_DIR="$(cd "${TEST_DIR}/../.." || exit 1; pwd -P)"
+TEST_DIR="$(
+  cd "$(dirname "${BATS_TEST_FILENAME}")" || exit 1
+  pwd -P
+)"
+ROOT_DIR="$(
+  cd "${TEST_DIR}/../.." || exit 1
+  pwd -P
+)"
 # detect ROOT_DIR - end
 
 setup() {
